go.mod

```diff
@@ -5,6 +5,7 @@ require (
 	github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc // indirect
```

Comment:
Was wondering why you committed this? You must have reason

Reply:
I had used logrus. However, I removed it as no longer in use. :)